modules/efk-logging-deploy-storage-considerations.adoc

```diff
@@ -84,7 +84,14 @@ absolute storage consumption around 50% and below 70% at all times]. This
 helps to avoid Elasticsearch becoming unresponsive during large merge
 operations.
 
-At 85% ES stops allocating new data to the node, at 90% ES starts de-allocating 
+By default, at 85% ES stops allocating new data to the node, at 90% ES starts de-allocating 
 existing shards from that node to other nodes if possible. But if no nodes have 
 free capacity below 85% then ES will effectively reject creating new indices 
-and becomes RED.
\ No newline at end of file
+and becomes RED. 
+
+[NOTE]
+====
+These low and high watermark values are Elasticsearch defaults. You can modify these values, 
+but you will also need to apply any modifications to the alerts also. The alerts are based 
+on these defaults.
```

Comment:
Can we say "for now"?
We already have solution for this but it will come in next releases.